### PR TITLE
Add NLog SequenceId to ECS documents

### PR DESCRIPTION
**Pull request by Mpdreamz**

### CODE REVIEW

File: src/Elastic.CommonSchema.NLog/EcsLayout.cs [hunk 1]
```diff
@@ -539,6 +539,7 @@ private Event GetEvent(LogEventInfo logEventInfo)
 					? long.Parse(eventSeverity)
 					: GetSysLogSeverity(logEventInfo.Level),
 				Timezone = TimeZoneInfo.Local.StandardName,
+				Sequence = logEventInfo.SequenceID
```
Comment (snakefoot) [inline src/Elastic.CommonSchema.NLog/EcsLayout.cs:542]: Notice that SequenceID is an integer. Thus if exceeding more than  2,147,483,647 LogEvents, then you will get bad behavior (imagine a service running for several days). See also: https://github.com/NLog/NLog/issues/4159

Comment (snakefoot) [reply src/Elastic.CommonSchema.NLog/EcsLayout.cs:542]: Maybe consider using `Layout.FromMethod` just like for TraceId / SpanId.

Comment (Mpdreamz) [reply src/Elastic.CommonSchema.NLog/EcsLayout.cs:542]: yeah I noticed it was `int` 😢 I assumed NLog had a way of dealing with overflows.

But reading https://github.com/NLog/NLog/issues/4159 SequenceId is actually lazy now and only increments if a layout is requesting it correct?

What happens with folks using `SequenceId` today, most likely overflowing? 

Whats your guidance @snakefoot keep it in so we can move along when NLog fixes this?
Or keep it out till then?

Comment (snakefoot) [reply src/Elastic.CommonSchema.NLog/EcsLayout.cs:542]: I was thinking to obsolete NLog `LogEventInfo.SequenceID` and instead rely on LayoutRenderer like `${counter}`.

I would probably implement EcsLayout SequenceId like ProcessId has been implemented. By using a Layout, then people can do whatever they want.

Comment (snakefoot) [reply src/Elastic.CommonSchema.NLog/EcsLayout.cs:542]: Alternative just have a private long-field, and do Interlocked.Increment in the NLog EcsLayout for every LogEvent (Skip the overhead of NLog Layout)

But I don't know how Elastic Sequence are expected to be filled out when, when having single instance restarting frequently (starts counting from Sequence=0), or having multiple process-instances on the same machine.

Comment (Mpdreamz) [reply src/Elastic.CommonSchema.NLog/EcsLayout.cs:542]: Okay in that case will close this! Thanks @snakefoot

**Pull request closed without merging**